shared/styles/index.native.tsx

```diff
@@ -40,9 +40,9 @@ const util = {
     backgroundColor: globalColors.greyLight,
     height: 16,
   },
-  mediumSubNavWidth: isTablet ? '25%' : '100%',
```

Comment:
we should never use percents in these types of shared values as its entirely dependant on the view hierarchy